cypress/integration/pages/about.ts

```diff
@@ -47,21 +47,11 @@ describe('/about', () => {
         text: 'Submit an issue ',
         href: 'https://github.com/iron-fish/ironfish/issues',
       },
-      {
```

Comment:
what's happening here? why was this removed? it sounds like this PR's goal is to introduce snapshot testing, and this change makes it look like other changes were made.